administrator/models/user_profile.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+"""
+Allows to create a user in a database.
+
+Contains a model class for users and a manager for this model.
+
+:copyright: (c) 2017 by Ol'ha Leskovs'ka
```

Comment:
Probably you shouldn't leave your copyright here, because it's our team project.